tests/integration_tests/test_rm/test_lsf.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# pylint: disable=protected-access, unused-argument, no-value-for-parameter
+
+import os
+
+from unittest import mock, TestCase
+
+import radical.utils as ru
+
+from radical.pilot.agent.resource_manager.lsf_summit import LSF_SUMMIT
+
+
+# ------------------------------------------------------------------------------
+#
+class TestTask(TestCase):
+
+    # --------------------------------------------------------------------------
+    #
+    @classmethod
+    def setUpClass(self) -> dict:
+        path = os.path.dirname(__file__) + '/../test_config/resources.json'
+        resources = ru.read_json(path)
+        hostname = 'summit'
```

Comment:
here is the same as for jsrun test, everything else looks good!
```python
    @classmethod
     def setUpClass(cls):
         path = os.path.dirname(__file__) + '/../test_config/resources.json'
         resources = ru.read_json(path)

         cls.host = 'summit'
         cls.resource = resources[cls.host]
```
btw, if `cls.host` is not used later then no need to keep this class attribute